TFNetwork.py

```diff
@@ -696,6 +705,12 @@ def get_layer(self, layer_name):
       if not self.parent_net:
         raise LayerNotFound("cannot get layer %r, no parent net for %r" % (layer_name, self))
       return self.parent_net.get_layer(layer_name[len("base:"):])
+    if '/' in layer_name:
```

Comment:
This logic should not be in `construct_layer`, but in `get_layer`.

Reply:
I guess you're talking about line 391 to 396, right? 
Until now, the `get_layer` method is not performing/ calling any construction. Do you really want to change that?

Reply:
Maybe this was moved. I think it was in `LayerBase.construct_layer` before. Now it's in `LayerBase.get_layer`, which is what I wanted.